config/default/GUIDE.md

```diff
@@ -18,7 +18,7 @@ To perform individual actions, see [medic-conf's documentation](https://github.c
 
 ## Modifying Forms
 
-Generally, the forms are created and edited in XLSX format (a spreadsheet, openable with Excel, Libreoffice or your favourite office tool) and later converted into XML format to upload to the instance. The forms are categorized into app forms (action/task forms such as pregnancy registration, delivery report) and contact forms (place/person forms).
+Generally, the forms are created and edited in XLSX format (a spreadsheet format that can be opened with Excel, Libreoffice, Google Sheets, or your favourite Office tool) and later converted into a XML format to upload to the instance. The forms are categorized into app forms (Action or Task forms such as pregnancy registration, delivery report) and contact forms (Place or Person forms).
```

Comment:
Wouldn't "an XML" would be correct, since it's not how it's spelt but how it sounds ("ex-em-el").

I'm uncomfortable capitalising and splitting into "Action or Task" is not correct, since capitalisation implies that it's a proper noun or otherwise named thing, which isn't really the case and hasn't been used elsewhere that I'm aware of.

Reply:
Will update to "an XML".

As for the Action or Task form, it is the language used in docs.communityhealthtoolkit.org's [ANC Reference App Overview](https://docs.communityhealthtoolkit.org/resource-overviews/anc-reference-app-overview.pdf) (see the Form Hierarchy slide) so think it is fine even though it isn't codified in the app. I also felt uncomfortable with it at first since it wasn't a true distinction in the app, but since app designers have been referencing forms as such I am supportive of using it here. I will change to lower case for now and we can revise later if needed.

Reply:
```suggestion
Generally, each form is created and edited as an [XLSForm](http://xlsform.org) file, which is an Excel spreadsheet file that uses a special notation, and can also be opened with Libreoffice, Google Sheets, or your favourite Office tool. The XLSForm is later converted into an [XForm](https://opendatakit.github.io/xforms-spec/) XML file to upload to the instance. The forms are categorized into app forms (action or task forms such as pregnancy registration, delivery report) and contact forms (Place or Person forms).
```